acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -32,20 +30,10 @@
 
 install_packages_on(hosts, PACKAGES)
 
+# make sure install is sane, beaker has already added puppet and ruby
+# to PATH in ~/.ssh/environment
 agents.each do |agent|
-  case agent['platform']
-  when /windows/
-    arch = agent[:ruby_arch] || 'x86'
-    base_url = ENV['MSI_BASE_URL'] || "http://builds.puppetlabs.lan/puppet-agent/#{ENV['SHA']}/artifacts/windows"
```

Comment:
So I presume there's still an equivalent way to use the ENV vars to install from custom builds?

It doesn't look that `download_url` is overridable in Beaker itself...  but maybe that's obsolete now?

https://github.com/puppetlabs/beaker/blob/d9a52536ed86bdf0ff544d50febe95d79912b95f/lib/beaker/dsl/install_utils/foss_utils.rb#L1094